dashboard/app/models/user.rb

```diff
@@ -441,7 +441,7 @@ def email_for_enrollments
       status: 'accepted'
     ).order(application_year: :desc).first&.form_data_hash
     alternate_email = latest_accepted_app ? latest_accepted_app['alternateEmail'] : ''
-    alternate_email&.empty? ? email : alternate_email
```

Comment:
I found a bug here where if `alternate_email` is `nil`, it will get caught by the safe chaining first (then defaulting to `alternate_email`) before `empty?` can be called on it.